src/main/java/com/auctionapp/api/controller/BidController.java

```diff
@@ -32,13 +32,27 @@ public BidController(final BidService bidService) {
 		return ResponseEntity.status(HttpStatus.OK).body(bids);
 	}
 
+	@GetMapping("/bidder/{bidderId}")
+	public ResponseEntity<List<BidDto>> getAuctionsByBidder(@PathVariable final UUID bidderId) {
+		final List<BidDto> bids = service.getAuctionsByBidder(bidderId);
+
+		return ResponseEntity.status(HttpStatus.OK).body(bids);
+	}
```

Comment:
This does not return auctions? I know that they are contained within bids, however, despite that, this method does not do what it says it does, and also as one user can place multiple bids on an auction they will not necessarily be distinct. You should be able to ask the auction service to give you all the distinct auctions where a certain user placed a bid if that's what you're aiming for here.